BUILD_OS_IMAGE.md

```diff
@@ -180,7 +180,17 @@ This can be achieved as follows:
 
     * Set it to off.
 
-* Shut down without logging out.
+* Enable autologon by creating or editing the following registry values (all of type REG_SZ):
```

Comment:
No preparation needed to make it work on Linux?

Reply:
We don't need to log in on Linux. We're running it as a systemd (non-user) service there. On Windows, we want to run the test runner in a normal user session rather than a service, which means we need to login.